### ISSUE TITLE
Screen flickering in Mario 3d Land Citra mmj (Unisoc T606 version)

### ISSUE THREAD

**Issue by Danixcrack2005** [weihuoya/citra#857]
In the Mario 3D Land title the screen flickers too much, my cell phone's processor is Unisoc T606, I use an Infinix Hot 30 i, please fix that error in the next update

**Comment by Danixcrack2005** [comment]
Uploading VID-20231012-WA0009.mp4…

**Comment by Kronous1st** [comment]
pro tips, buy a new device with better Processor and GPU

**Comment by Ishan0981** [comment]
> pro tips, buy a new device with better Processor and GPU

This Is A Emulator Problem Not Device Because I Also Have The Problem Because Citra MMJ Is Not Compatible With Mario Games

**Comment by Danixcrack2005** [comment]
Exactly, it's not the processor's fault, it's the emulator's fault since I have another device and the same thing happens
